chat-plugins/info.js

```diff
@@ -1993,363 +1953,33 @@ var commands = exports.commands = {
-		}
-		if (target === 'makechatroom') {
-			matched = true;
-			this.sendReply("/makechatroom [roomname] - Creates a new room named [roomname]. Requires: ~");
-		}
-		if (target === 'deregisterchatroom') {
-			matched = true;
-			this.sendReply("/deregisterchatroom [roomname] - Deletes room [roomname] after the next server restart. Requires: ~");
-		}
-		if (target === 'roomowner') {
-			matched = true;
-			this.sendReply("/roomowner [username] - Appoints [username] as a room owner. Removes official status. Requires: ~");
-		}
-		if (target === 'roomdeowner') {
-			matched = true;
-			this.sendReply("/roomdeowner [username] - Removes [username]'s status as a room owner. Requires: ~");
-		}
-		if (target === 'privateroom' || target === 'hiddenroom') {
-			matched = true;
-			this.sendReply("/privateroom [on/off] - Makes or unmakes a room private. Requires: ~");
-			this.sendReply("/hiddenroom [on/off] - Makes or unmakes a room hidden. Hidden rooms will maintain global ranks of users. Requires: \u2605 ~");
-		}
+	showtan: function (target, room, user) {
+		if (room.id !== 'showderp') return this.sendReply("The command '/showtan' was unrecognized. To send a message starting with '/showtan', type '//showtan'.");
+		if (!this.can('modchat', null, room)) return;
+		target = this.splitTarget(target);
+		if (!this.targetUser) return this.sendReply("User not found");
+		if (!room.users[this.targetUser.userid]) return this.sendReply("Not a showderper");
+		this.targetUser.avatar = '#showtan';
+		room.add("" + user.name + " applied showtan to affected area of " + this.targetUser.name);
+	},
 
-		// overall
-		if (target === 'help' || target === 'h' || target === '?' || target === 'commands') {
-			matched = true;
-			this.sendReply("/help OR /h OR /? - Gives you help.");
-		}
-		if (!target) {
-			this.sendReply("COMMANDS: /nick, /avatar, /rating, /whois, /msg, /reply, /ignore, /away, /back, /timestamps, /highlight");
-			this.sendReply("INFORMATIONAL COMMANDS: /data, /dexsearch, /movesearch, /groups, /opensource, /avatars, /faq, /rules, /intro, /tiers, /othermetas, /learn, /analysis, /calc (replace / with ! to broadcast. Broadcasting requires: + % @ & ~)");
-			if (user.group !== Config.groupsranking[0]) {
-				this.sendReply("DRIVER COMMANDS: /warn, /mute, /unmute, /alts, /forcerename, /modlog, /lock, /unlock, /announce, /redirect");
-				this.sendReply("MODERATOR COMMANDS: /ban, /unban, /ip");
-				this.sendReply("LEADER COMMANDS: /declare, /forcetie, /forcewin, /promote, /demote, /banip, /unbanall");
-			}
-			this.sendReply("For an overview of room commands, use /roomhelp");
-			this.sendReply("For details of a specific command, use something like: /help data");
-		} else if (!matched) {
-			this.sendReply("Help for the command '" + target + "' was not found. Try /help for general help");
-		}
+	cpgtan: function (target, room, user) {
```

Comment:
I agree, I just didn't know if they should each have their own file to follow the pattern for the other chat-plugin files or if they should be put in some generic "other" file because there is so little content to them.